methods/methods_release2.md

```diff
@@ -93,49 +95,262 @@ Demultiplexing, feature-table generation, fragment insertion, and taxonomic prof
+  --p-min-feature-frequency 10 \
+  --p-min-sample-count 0 \
+  --o-biplot emp500_18s_deblur_merged_biom_euks_noSingletons_noControls_min5500_rpca_pca.qza \
+  --o-distance-matrix emp500_18s_deblur_merged_biom_euks_noSingletons_noControls_min5500_rpca_dist.qza`
 
 ### 1.3 Fungal ITS data
 
 #### Demultiplexing:
-* Process: Split libraries FASTQ
-** Parameters: Multiplexed FASTQ, Golay 12 base pair reverse complement mapping file barcodes
+`qiime demux emp-paired \
+  --i-seqs emp-paired-end-sequences.qza \
+  --m-barcodes-file emp500_prep_info_ITS.txt \
+  --m-barcodes-column 'barcode' \
+  --p-rev-comp-mapping-barcodes \
+  --o-error-correction-details emp500_its_demux.qza \
+  --o-per-sample-sequences emp500_its_demux.qza`
+
+#### Trimming primers:
+`qiime cutadapt trim-paired \
+  --i-demultiplexed-sequences emp500_its_demux.qza \
+  --p-adapter-f GCATCGATGAAGAACGCAGC \
+  --p-front-r GCTGCGTTCTTCATCGATGC \
+  --o-trimmed-sequences emp500_its_demux_trimmed.qza`
+  
+#### Export data and process for denoising with Deblur (standalone)
+Note: The sample list is a text file where each row is a unique sample identifier that corresponds to your sequence file names. For example, 'uren.s001' corresponds to the sequence files 'uren.s001_S1_L001_R1.fastq.gz' and 'uren.s001_S1_L001_R2.fastq.gz'.
 
-#### Sequence trimming, denoising, feature-table generation, and fragment insertion:
-* Process: Trimming
-* Parameters: 150 base pair
+`qiime tools export \
+  --input-path emp500_its_demux_trimmed.qza \
+  --output-path emp500_its_demux_trimmed`
+  
+`input_path='emp500_its_demux_trimmed/'`
+`output_path='emp500_its_demux_trimmed_merged_files/'`
+`sample_list='sample_list.txt'`
 
-* Process: Deblur
-* Parameters: Default settings
+`for i in $(cat < "$sample_list");
+do
+cat "$input_path""$i"*R1*fastq.gz "$input_path""$i"*R2*fastq.gz > "$output_path""$i".fastq.gz
+done`
 
-#### Taxonomic profiling
-* Process: Feature-classifier sklearn 
-* Parameters: Using the UNITE 8 release as a reference
+#### Denoising
+`deblur workflow \
+  --seqs-fp emp500_its_demux_trimmed_merged_files/ \
+  --output-dir emp500_its_deblur/ \
+  --trim-length 150 \
+  --threads-per-sample 0 \
+  --overwrite`
 
-#### OTU clustering
-* Process: Closed-reference OTU picking 
-* Parameters: 97% sequence similarity threshold, using the UNITE 8 release as a reference
+Import denoised sequences and feature-table into QIIME 2
+
+`qiime tools import \
+  --input-path emp500_its_deblur/all.biom \
+  --type 'FeatureTable[Frequency]' \
+  --input-format BIOMV210Format \
+  --output-path emp500_its_deblur/all_biom.qza`
+
+`qiime tools import \
+  --input-path emp500_its_deblur/all.seqs.fa \
+  --output-path emp500_its_deblur/all_seqs.qza \
+  --type 'FeatureData[Sequence]'`
+  
+At this point denoised feature-tables and sequences from each sequencing lane were merged.
 
+#### Merging tables:
+`qiime feature-table merge \
+  --i-tables emp500_its_deblur_prep_8702_biom.qza \
```

Comment:
same q here as in the 18S section

Reply:
Thanks! I believe the issues is resolved.